src/limb_repo/approaches/lr_system.py

```diff
@@ -0,0 +1,26 @@
+"""A base class defining the API for a Limb Repositioning System."""
+
+import abc
+
+from limb_repo.environments.base_env import BaseEnv
+from limb_repo.structs import Controller, State
+
+
+class LimbRepositioningSystem:
```

Comment:
I'm wondering if this whole class and file is actually needed -- maybe it's an unnecessary abstraction?

Reply:
yeah I'll delete it for now